packages/salesforce-adapter/src/adapter_creator.ts

```diff
@@ -300,6 +303,14 @@ export const adapter: Adapter = {
       config: config[CLIENT_CONFIG],
     })
     let deployProgressReporterPromise: Promise<DeployProgressReporter> | undefined
+    let allElements: Element[] | undefined
+
+    const getAllElements = async (): Promise<Element[]> => {
+      if (_.isUndefined(allElements)) {
```

Comment:
```suggestion
      if (allElements === undefined) {
```
No need for lodash functions when simple syntax will do just as well.